webapp/src/index.tsx

```diff
@@ -29,15 +29,25 @@ export default class Plugin {
             );
 
             registry.registerRootComponent(BulkAddChannelModal);
+
+            this.setupUIFinished = true;
         };
 
-        registry.registerRootComponent(() => <SetupUI setup={setup}/>);
+        registry.registerReducer(reducers);
+        registry.registerRootComponent(() => (
+            <SetupUI
+                setup={setup}
+                setupUIFinished={this.setupUIFinished}
+            />
+        ));
     }
 }
 
-const SetupUI = ({setup}: { setup: () => Promise<void> }) => {
+const SetupUI = ({setup, setupUIFinished}: { setup: () => Promise<void>, setupUIFinished: boolean }) => {
     useEffect(() => {
-        setup();
+        if (!setupUIFinished) {
+            setup();
+        }
     }, []);
```

Comment:
Under what condition does this "root component" get mounted twice? The solution in the PR seems fine if it fixes the problem, but I'd like to understand what causes the problem

Reply:
When you go to the system console and come back this get called again, adding another "Bulk invite" option to the channel header menu. --> #9 

I checked how they did it in the Jira plugin.